Wabbajack.App.Wpf/Util/SystemParametersConstructor.cs

```diff
@@ -124,18 +124,24 @@ public SystemParameters Create()
 
         private string GetGPUName()
         {
+            var gpuManufacturers = new [] {"amd", "intel", "nvidia"};
```

Comment:
Any particular reason to filter on these? Would also use a HashSet here for faster lookups. Even if it's a super small array, it's good practice when you want to have a list that you're gonna do lots of Contains on